src/components/RedoIcon.tsx

```diff
@@ -4,19 +4,43 @@ import { token } from '../../styled-system/tokens'
 import Icon from '../@types/Icon'
 
 /** A redo icon. */
-const RedoIcon = ({ fill, size = 18, style, className }: Icon) => {
+const RedoIcon = ({ fill, size = 14, style = {}, className }: Icon) => {
+  const scalingFactor = 1.37
```

Comment:
If the scaling factor is the same for all icons, we should probably add it to `constants.ts` rather than repeating it each time.

Reply:
Based on the current set of replaced SVGs, the scaling factor is indeed the same for all icons, though this wasn't clear before the replacements, and it’s still unknown for other icons. We can move the scaling factor to constants.ts, and if a different factor is needed for a specific icon later on, it can be overridden by defining a new one instead of importing from constants. So yes, let's move it to constants.

Reply:
Scaling factor has been moved to constants.